idaes/models_extra/gas_solid_contactors/unit_models/fixed_bed_1D.py

```diff
@@ -0,0 +1,2047 @@
+            for j in component_list:
+                break
+            for (t, x), c in self.solid_sum_component_eqn.items():
+                iscale.constraint_scaling_transform(
+                    c,
+                    iscale.get_scaling_factor(
+                        self.solid_properties[t, x].mass_frac_comp[j]
+                    ),
+                    overwrite=False,
+                )
+
+        if self.config.energy_balance_type != EnergyBalanceType.none:
+
+            if hasattr(self, "solid_energy_holdup_calculation"):
+                for (t, x), c in self.solid_energy_holdup_calculation.items():
+                    sf1 = iscale.get_scaling_factor(self.solid_phase_area[t, x])
+                    sf2 = iscale.get_scaling_factor(
+                        self.solid_properties[t, x].get_energy_density_terms("Sol")
+                    )
+                    iscale.constraint_scaling_transform(c, sf1 * sf2, overwrite=False)
+
+            if hasattr(self, "solid_enthalpy_balances"):
+                for (t, x), c in self.solid_enthalpy_balances.items():
+                    sf1 = iscale.get_scaling_factor(self.solid_phase_area[t, x])
+                    sf2 = iscale.get_scaling_factor(
+                        self.solid_properties[t, x].enth_mass
+                    )
+                    iscale.constraint_scaling_transform(
+                        c, 1e-2 * sf1 * sf2, overwrite=False
+                    )
+            if hasattr(self.gas_phase, "enthalpy_balances"):
+                phase_list = self.gas_phase.properties.phase_list
+                for (t, x), c in self.gas_phase.enthalpy_balances.items():
+                    sf = iscale.min_scaling_factor(
+                        [self.gas_phase._enthalpy_flow[t, x, p] for p in phase_list]
+                    )
+                    iscale.constraint_scaling_transform(c, 1e-3 * sf, overwrite=True)
+
+            if hasattr(self, "solid_energy_accumulation_disc_eq"):
+                for (t, x), c in self.solid_energy_accumulation_disc_eq.items():
+                    iscale.constraint_scaling_transform(
+                        c,
+                        1e-3
+                        * iscale.get_scaling_factor(
+                            self.solid_energy_accumulation[t, x]
+                        ),
+                        overwrite=False,
+                    )
+
+            if hasattr(self.gas_phase, "energy_accumulation_disc_eq"):
+                for (t, x, p), c in self.gas_phase.energy_accumulation_disc_eq.items():
+                    iscale.constraint_scaling_transform(
+                        c,
+                        1e-3
+                        * iscale.get_scaling_factor(
+                            self.gas_phase.energy_accumulation[t, x, p]
+                        ),
+                        overwrite=True,
+                    )
+
+    def _get_stream_table_contents(self, time_point=0):
```

Comment:
It would be good to also define the `_get_performance_contents` method.

Reply:
Done.